src/Core/AdminConsole/OrganizationFeatures/OrganizationUsers/InviteUsers/Validation/GlobalSettings/EnvironmentValidator.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Bit.Core.AdminConsole.Shared.Validation;
+
+namespace Bit.Core.AdminConsole.OrganizationFeatures.OrganizationUsers.InviteUsers.Validation.GlobalSettings;
+
+public interface IEnvironmentValidator : IValidator<EnvironmentRequest>;
+
+public class EnvironmentValidator : IEnvironmentValidator
+{
+    public async Task<ValidationResult<EnvironmentRequest>> ValidateAsync(EnvironmentRequest value) =>
```

Comment:
See the many annotations here in the Github diff:

> This async method lacks 'await' operators and will run synchronously. Consider using the 'await' operator to await non-blocking API calls, or 'await Task.Run(...)' to do CPU-bound work on a background thread.

I think that `async` is a convenient way to wrap the return result in a Task, but I guess the compiler disagrees. You may have to do it explicitly.